desktop/app-handlers/crash-reporting/index.js

```diff
@@ -11,10 +11,10 @@ const debug = require( 'debug' )( 'desktop:crash-reporting' );
 /**
  * Internal dependencies
  */
-const Config = require( 'lib/config' );
+const Config = require( '../../lib/config' );
 
 module.exports = function() {
-	if ( Config.crash_reporter.electron ) {
+	if ( Config.crash_reporter && Config.crash_reporter.electron ) {
```

Comment:
> I recall seeing errors from this line in my recent attempts to merge desktop > calypso so this is nice to see.

We were probably getting the error for the same reason: the wrong config was being imported.  I'm hoping in the end this change isn't necessary